### DISCUSSION TITLE
Dependency Graph not showing any dependents when there should be

### DISCUSSION THREAD

**Discussion by cwendt94** [community/community#21637] [New to GitHub]
<p>My repo is only showing dependencies and not dependents. Another repo shows in their dependency referencing my repo but it is not showing up as a dependent on mine.</p>
<p>Here is my repo expecting dependents <a href="https://github.com/cwendt94/espn-api/network/dependents" class="inline-onebox" rel="noopener nofollow ugc">Network Dependents · cwendt94/espn-api · GitHub</a>.</p>
<p>Here is a repo referencing my package and shows in their dependency graph <a href="https://github.com/dtcarls/fantasy_football_chat_bot/network/dependencies" class="inline-onebox" rel="noopener nofollow ugc">Dependencies · dtcarls/fantasy_football_chat_bot · GitHub</a></p>

**Comment by sando240** [comment]
<p>I also want same Graph for my site result history how it go up or down etc.<br>
I have a site in which teer result game got updated daily. <a href="https://teerresultinfo.com/" rel="noopener nofollow ugc">https://teerresultinfo.com/</a><br>
<a href="https://teerresultinfo.com/" rel="noopener nofollow ugc"></a><br>
I also maintain history of each result. I want to show in graph.</p>

**Comment by ernest-phillips** [comment]
<p>Hi <a class="mention" href="/cwendt94">@cwendt94</a> :wave:</p>
<p>Can you link me to where your project is showing in the PIP package registry?</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Nevermind my last message. It looks like we needed to redetect the assignment to your repository. You should see the dependents now.</p>

**Comment by cwendt94** [comment]
<p>Thank you <a class="mention" href="/ernest-phillips">@ernest-phillips</a>, that worked!</p>
